pygments/lexers/qlik.py

```diff
@@ -0,0 +1,104 @@
+"""
+    pygments.lexers.qlik
+    ~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the qlik scripting language
+
+    :copyright: Copyright 2006-2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import RegexLexer, include, bygroups
+from pygments.token import *
+
+__all__ = ["QlikLexer"]
+
+
+class QlikLexer(RegexLexer):
+    """
+    Lexer for qlik code, including .qvs files
+    """
+
+    name = "Qlik"
+    aliases = ["qlik", "qlikview", "qliksense", "qlikscript"]
+    filenames = ["*.qvs", "*.qvw"]
+
+    flags = re.IGNORECASE
+
+    tokens = {
+        "comment": [
+            (r"[^*/]", Comment.Multiline),
+            (r"/\*", Comment.Multiline, "#push"),
+            (r"\*/", Comment.Multiline, "#pop"),
+            (r"[*/]", Comment.Multiline),
+        ],
+        "numerics": [
+            (r"\b[0-9][0-9]*\.[0-9]+([eE][0-9]+)?[fd]?\b", Number.Float),
+            (r"\b[0-9]+\b", Number.Integer),
+        ],
+        "variables": [
+            (r"(\$\()(\w+)(\))", bygroups(Keyword, Name.Variable, Keyword)),
+        ],
+        "root": [
+            (r"\s+", Text),
```

Comment:
Isn't that https://pygments.org/docs/tokens/?

Reply:
Thanks @Jean-Abou-Samra . Not sure if it's been written since I opened this last year, or if I was just looking in the wrong place. Very useful though!

Reply:
Commit history tells me that it has been around for a long time.

Reply:
Clearly just looking in the wrong place then, or not looking for the right thing. Thanks!